docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -1,332 +1,407 @@
+3. From the priority list, select the first device capable of supporting the given precision. 
+4. If the network’s precision is FP32 but there is no device capable of supporting it, offload the network to a device supporting FP16. 
 
-@snippet snippets/AUTO2.cpp part2
+@sphinxdirective
++----------+-------------------------------------------------+-------------------------------------+
+| Choice   | | Supported                                     | | Supported                         |
+| Priority | | Device                                        | | model precision                   |
++==========+=================================================+=====================================+
+| 1        | | dGPU                                          | FP32, FP16, INT8, BIN               |
+|          | | (e.g. Intel® Iris® Xe MAX)                    |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+| 2        | | VPUX                                          | INT8                                |
+|          | | (e.g. Intel® Movidius® VPU 3700VE)            |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+| 3        | | iGPU                                          | FP32, FP16, BIN,                    |
+|          | | (e.g. Intel® UHD Graphics 620 (iGPU))         |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+| 4        | | Myriad                                        | FP16                                |
+|          | | (Intel® Neural Compute Stick 2 (Intel® NCS2)) |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+| 4        | | IA CPU                                        | FP32, FP16, INT8, BIN               |
+|          | | (e.g. Intel® Core™ i7-1165G7)                 |                                     |
++----------+-------------------------------------------------+-------------------------------------+
+@endsphinxdirective
 
-You can also use the Auto-device plugin to choose a device from a limited choice of devices, in this example CPU and GPU:
+To put it simply, when loading the network to the first device on the list fails, AUTO will try to load it to the next device in line, until one of them succeeds. For example: 
+If you have dGPU in your system, it will be selected for most jobs (first on the priority list and supports multiple precisions). But if you want to run a WINOGRAD-enabled IR, your CPU will be selected (WINOGRAD optimization is not supported by dGPU). If you have Myriad and IA CPU in your system, Myriad will be selected for FP16 models, but IA CPU will be chosen for FP32 ones.  
```

Comment:
1) What is "IA CPU"?
2) Please use the correct name for Myriad: an Intel® Movidius™ Myriad™ X VPU?

Reply:
I assume IA describes a type, or architecture of a CPU - it was added by developers specifically (will be revised in further reviews). When it goes to "Myriad" in this case, it is a direct reference to the table, so I'm not sure the full name needs to be used. If you think it does, I won't argue though.